test/Unit/Api/ClientTest.php

```diff
@@ -274,7 +274,7 @@ public function testGettingASpecificResourceReturnsACollectionOfThatResource($pa
      */
     public function testGettingTheCountOfACollectionReturnsThatCollectionsCount($path, $fnName, $class)
     {
-        if (in_array($path, array('coupons', 'orderstatuses', 'products/skus', 'requestlogs'))) {
+        if (in_array($path, array('coupons', 'order_statuses', 'products/skus', 'requestlogs'))) {
             $this->markTestSkipped(sprintf('Getting the count The php client does not support getting the count of %s', $path));
```

Comment:
Hmm, I just now noticed this, would you mind fixing the grammar here? It should read `The PHP client does not support getting the count of %s`.